app/domain/output_document.rb

```diff
@@ -3,10 +3,84 @@ class OutputDocument
 
   attr_accessor :appointment_summary
 
+  delegate :id, :guider_name, :income_in_retirement, :continue_working, :unsure,
+           :leave_inheritance, :wants_flexibility, :wants_security,
+           :wants_lump_sum, :poor_health,
+           to: :appointment_summary
+
+  delegate :address_line_1, :address_line_2, :address_line_3, :town, :county,
+           :postcode,
+           to: :appointment_summary, prefix: :attendee
+
   def initialize(appointment_summary)
     @appointment_summary = appointment_summary
   end
 
+  def attendee_name
+    "#{appointment_summary.title} #{appointment_summary.first_name} #{appointment_summary.last_name}".squish
+  end
+
+  def appointment_date
+    appointment_summary.date_of_appointment.to_s(:long)
+  end
+
+  def guider_organisation
+    if appointment_summary.guider_organisation == 'tpas'
+      'The Pensions Advisory Service'
+    else
+      'Pension Wise'
+    end
+  end
+
+  def value_of_pension_pots
```

Comment:
This needs updating to take into consideration whether the value given is an approximation.

If `value_of_pension_pots_is_approximate == true` then the output should read `£50,000 (approximately)`. If `!upper_value_of_pension_pots.blank?`, then ignore `value_of_pension_pots_is_approximate`.